qa/TL0_FW_iterators/test_jax.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 # used pip packages
-pip_packages='${python_test_runner_package} numpy jax clu'
+pip_packages='${python_test_runner_package} numpy<2 jax clu'
```

Comment:
I think https://github.com/NVIDIA/DALI/blob/main/qa/setup_packages.py#L479 should be enough.